src/components/MainMemo/MemoButton/style.ts

```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components';
+
+import * as T from './type';
+
+export const Button = styled.button<T.StyledButton>`
+  align-items: center;
+  width: 90px;
+  height: 30px;
+  border-radius: 5px;
+  border: 1px solid #9ca3af;
+  font-weight: bold;
+  font-size: 13px;
+  text-align: center;
+  background-color: ${(props) => props.theme.background[props.variant]};
+  color: ${(props) => props.theme.text};
+  &:active {
```

Comment:
```suggestion
  color: ${(props) => props.theme.text};
  
  &:active {
```
중괄호 앞에서는 한 줄 띄어주면 더 읽기 편해집니당 :)